scrapy/pipelines/files.py

```diff
@@ -269,24 +269,19 @@ class FilesPipeline(MediaPipeline):
 
     MEDIA_NAME = "file"
     EXPIRES = 90
-    STORE_SCHEMES = {
```

Comment:
I wonder if that's possible to have backwards compatibility layer for this; currently the change breaks any pipeline subclasses which override STORE_SCHEMES.